test/Toggle/IsCurrentUserToggleActiveTest.php

```diff
@@ -121,6 +136,65 @@ public function WhenToggleInvisibleAndNotEnabledByPolicyButReleaseDateIsTodayOrI
     }
 
     /**
+     * @test
+     */
+    public function GivenToggleInvisibleAndNotSetByPolicyAndReleaseDateIsInTheFutureAndUserIsAutoSubscribed_ThenInactive()
+    {
+        $this->setupChecker( false, false, false, false, false, false, false );
```

Comment:
I'm not 100% sure, but probably it wouldn't take more than an hour. I'll take a look at this tomorrow.